website/docs/docs/cloud/migration.md

```diff
@@ -7,34 +7,45 @@ pagination_next: null
 pagination_prev: null
 ---
 
-dbt Labs is in the process of migrating dbt Cloud to a new _cell-based architecture_. This architecture will be the foundation of dbt Cloud for years to come, and will bring improved scalability, reliability, and security to all customers and users of dbt Cloud.
+dbt Labs is in the process of rolling out a new cell-based architecture for dbt Cloud. This architecture will be the foundation of dbt Cloud for years to come, and brings improved reliability, performance, and consistency to users of dbt Cloud.
 
-There is some preparation required to ensure a successful migration.
+Migrations are being scheduled on a per-account basis. If you have yet to receive any communication (either with a banner or by email) about a migration date, you don't need to take any action at this time. dbt Labs will share information about your migration with you, with appropriate advance notice, when applicable to your account.
```

Comment:
Not sure if legal needs to look at this, but I suggest we more directly say that we're sending a communication.
```suggestion
We're scheduling migrations by account. When we're ready to migrate your account, you will receive a banner or email communication with your migration date. If you have not received this communication, then you don't need to take action at this time. dbt Labs will share information about your migration with you, with appropriate advance notice, when applicable to your account.
```

Reply:
Not a legal doc, so they didn't need the review but the language looks good